docs/lake/api.md

```diff
@@ -85,12 +87,12 @@ PUT /pool/{pool}
 | ---- | ---- | -- | ----------- |
 | pool | string | path | **Required.** ID or name of the requested pool. |
 | name | string | body | **Required.** The desired new name of the pool. Must be unique to lake. |
+| Content-Type | string | header | [MIME type](#mime-types) of the request payload. |
 
 **Example Request**
 
 ```
 curl -X PUT \
-     -H 'Accept: application/json' \
```

Comment:
This was one I spotted along the way: Since the successful HTTP 204 for this endpoint has no response payload and errors would come back only as JSON, it seems like this was unnecessary.